lighthouse-core/audits/seo/tap-targets.js

```diff
@@ -302,8 +302,15 @@ class TapTargets extends Audit {
     const failingTapTargetCount = new Set(overlapFailures.map(f => f.tapTarget)).size;
     const passingTapTargetCount = tapTargetCount - failingTapTargetCount;
 
-    const score = tapTargetCount > 0 ? passingTapTargetCount / tapTargetCount : 1;
-    const displayValue = str_(UIStrings.displayValue, {decimalProportion: score});
+    let score = 1;
+    let passingTapTargetRatio = 1;
+    if (failingTapTargetCount > 0) {
+      passingTapTargetRatio = (passingTapTargetCount / tapTargetCount);
+      // If there are any failures then we don't want the audit to pass,
+      // so keep the score below 90.
+      score = Math.min(passingTapTargetRatio * 0.9, 0.89);
```

Comment:
do we need to do both? isn't just doing 1 of these sufficient?

Reply:
Yeah just `* 0.89` is enough.

I was trying to interpret [this](https://github.com/GoogleChrome/lighthouse/issues/7338#issuecomment-468723187) and thought maybe you meant that multiplying by 0.9 gives a smoother score or something.

> I'd say go with `* .9/Math.min(score, .89)/targets.length ? 0 : 1` if each and every failure should be looked at.

Reply:
oh yeah was just trying to suggest using one of those not in combination :)

Reply:
Ahh, it makes sense now 🙂